tests/test_exception.cpp

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
```

Comment:
Should probably be called "test_exceptions.cpp" (with an s) to match the text name.

Reply:
(also can live under tests/core/?)